src/esw/drive/odrive_control.py

```diff
@@ -600,21 +580,54 @@ def _update(self) -> None:
             self._bridge_on_event(ODriveEvent.ODRIVE_ERROR_EVENT)
 
 
+class Application(object):
```

Comment:
Looks good, I think this is better than the separate processes. We can even implement some failover logic in here later if we need it for each pair.